ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ExpertRowMapper.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.directionsquestionnaire.CCDExpertReportRow;
+import uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.ExpertReportRow;
+
+@Component
+public class ExpertRowMapper {
+
+    public CCDCollectionElement<CCDExpertReportRow> to(ExpertReportRow expertReportRow) {
```

Comment:
I think mappers should map from plain CCD objects to our POJOS, ie. `ExpertRowMapper` should map from `CCDExpertReportRow` to `ExpertReportRow` (and viceversa), leaving the `CCDCollectionElement` out of the mapper. But I'll defer to more expert CCD folks's opinion

Reply:
i like that and it would have been slightly easier too but i just followed existing code..

Reply:
return type should be `CCDExpertReportRow` instead of `CCDCollectionElement<CCDExpertReportRow>`